README.md

```diff
@@ -1 +1,67 @@
-# first-project
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# first-project
+
+> building my first project to test understanding of github flow.
+
+
+## Built With
+
+- html , css
+
+
+## Live Demo (if available)
+
+[Live Demo Link](https://mohamed20163858.github.io/first-project/)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/mohamed20163858)
+- Twitter: [@twitterhandle](https://twitter.com/mohamed20163858)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/mohamed-mohsen-685a4995/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- The MIT link is not working. The reason is that you do not have the MIT license file in your repo. You can get the MIT.md from this link: https://github.com/microverseinc/readme-template which you should personalize it to make it as your license.

Reply:
@aboongm yeah thanks a lot I already included an MIT license file inside my repo and personalized it you can check it out